definitions/private-law/json/CaseEventToComplexTypes/EditReturnedOrder/SDO/CaseEventToComplexTypes.json

```diff
@@ -0,0 +1,387 @@
+[
```

Comment:
SDO can not be drafter by solicitor, so do we need these configs, connsidering the size of  PRL ccd config?